### PR TITLE
build 2.4.1 ❄️

### PR DESCRIPTION
**Pull request by boldorider4**
- [PKG-3566]
- [Upstream requirements](https://github.com/ramonhagenaars/nptyping/blob/v2.4.1/dependencies/requirements.txt)

[PKG-3566]: https://anaconda.atlassian.net/browse/PKG-3566?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

Channel: Snowflake

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,36 +10,55 @@ source:
...
-    - python >=3.6
+    - python
     - pip
+    - wheel
+    - setuptools
   run:
-    - python >=3.6
+    - python
     - numpy
     - typish >=1.5.2
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
    - numpy>=1.20.0,<2.0.0
    - typing_extensions >=4.0.0,<5.0.0  # [py<310]
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -10,36 +10,55 @@ source:
...
     - nptyping
   commands:
     - pip check
+    # ignoring test file with missing packages, also skipping s390x since feedparser not available
+    - pytest --ignore=tests/test_beartype.py --ignore=tests/test_pyright.py -k "not ({{ tests_to_skip }})" tests  # [not s390x]
   requires:
     - pip
+    - pytest
+    - pandas
+    - black
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
```
It's redundant

File: recipe/meta.yaml [hunk 3]
```diff
@@ -10,36 +10,55 @@ source:
...
+    - feedparser    # [not s390x]
+    - isort
+    - mypy
+    - pylint
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
```
I think they are redundant as linters

**Review by danpetry — APPROVED** [review]
Based on @skupr-anaconda 's comments being addressed

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**